text_collector_examples/mellanox_hca_temp

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -eu
+
+# Script to read Mellanox HCA temperature using the Mellanox mget_temp_ext tool
+
+# Copyright 2018 The Prometheus Authors
+# Author: Jan Phillip Greimann <jan.greimann@profitbricks.com>
```

Comment:
I meant that this should go above or below the whole copyright block

Reply:
Ah, okay. Moved below the block.